src/lua/modules/game.cpp

```diff
@@ -663,6 +663,21 @@ int luaGameReload(lua_State* L)
 	return 1;
 }
 
+int luaGameGetPlayerRecord(lua_State* L)
+{
+	// Game.getPlayerRecord()
+	tfs::lua::pushNumber(L, g_game.getPlayerRecord());
+	return 1;
+}
+
+int luaGameSetPlayerRecord(lua_State* L)
+{
+	// Game.setPlayerRecord(record)
+	g_game.setPlayerRecord(tfs::lua::getNumber<uint32_t>(L, 1));
+	tfs::lua::pushBoolean(L, true);
+	return 1;
+}
+
```

Comment:
I don't think this belongs to C++. We might need to move the HTTP handlers to Lua to remove it though